### PR TITLE
fixed wrong tarot numbers

### PR DESCRIPTION
**Pull request by s1910238061**

### CODE REVIEW

File: angry-bot/src/data/tarot-texts.ts [hunk 1]
```diff
@@ -564,33 +564,33 @@ export const tarots: Tarot[] = [
         text: "Up until the 19th century, people used to be :angry all the time. But then they discovered that being angry all the time is good for your health. So they stopped being angry. And that is why not everybody is angry today. Pathetic.",
```
Comment (Woife5) [inline angry-bot/src/data/tarot-texts.ts:564]: We can mabe fix this `:angry` aswell or is it intended? :o

Comment (Woife5) [reply angry-bot/src/data/tarot-texts.ts:564]: or should that be `:angry191:`?

**Review by Woife5 — CHANGES_REQUESTED** [review]

File: angry-bot/src/data/tarot-texts.ts (hunk 1 above)
Comment (s1910238061) [reply angry-bot/src/data/tarot-texts.ts:564]: is not intended! Thanks!

Comment (s1910238061) [reply angry-bot/src/data/tarot-texts.ts]: fixed it in new push!

**Review by Woife5 — APPROVED** [review]

**Pull request merged**